UnityProject/Assets/Scripts/Core/Database/PersistenceServer.cs

```diff
@@ -59,7 +63,7 @@ public static async Task<ApiResult<AccountGetCharacterSheets>> GetAccountsCharac
 
 		public static async Task<ApiResult<JsonObject>> DeleteAccountsCharacterByID(int id, string token)
 		{
-			var response = await ApiServer.Delete<JsonObject>(GetUri($"characters/{id}/delete"), token);
+			var response = await ApiServer.Delete<JsonObject>(GetUri($"characters/{id}/delete",""), token);
```

Comment:
what's the extra parameter for?

Reply:
I checked it. It ensures no query is added to the URI. I imagine Max is explicity sending empty string because for some reason the uri was built with a bunch of extra query parameters that we don't need.